bundles/test/org.eclipse.smarthome.test/META-INF/MANIFEST.MF

```diff
@@ -12,8 +12,19 @@ Export-Package:
  org.eclipse.smarthome.test;uses:="org.osgi.framework"
 Import-Package: com.google.common.collect,
  groovy.lang,
+ javax.servlet,
+ javax.servlet.http,
  org.apache.commons.io,
  org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.jetty.client;version="[9.3.15,9.4.0)",

```

Comment:
Does it have to be this specific version?
If not, I'd prefer to remove this constraint as it means that it'll break if we one day switch to a more recent Jetty version in the TP.

Reply:
Unfortunately the version constraints are necessary. Otherwise the tests will run into massive dependency resolution failures during maven build.
I migrated the TestServer from the fsinternetradio binding where the constraints where already present.

I will reduce the given dependencies in the next commit since the client depds are not needed.